doc/docs/how-tos/err_cmd.md

```diff
@@ -0,0 +1,67 @@
+# Skip Failed Datums
+
+!!! note "Summary"
+    The `err_cmd` parameter enables you to fail a datum without failing the
+    whole job.
+
+!!! note
+    Before you read this section, make sure that you understand such
+    concepts as [Datum](../../concepts/pipeline-concepts/datum/) and
+    [Pipeline](../../concepts/pipeline-concepts/pipeline/).
+
+When Pachyderm processes your data, it breaks it up into units of
+computation called datums. Each datum is processed separately.
+In a basic pipeline configuration, a failed datum results in a failed
+job. However, in some cases, you might not need all datums
+to consider a job successful. If your downstream pipelines can be run
+on only the successful datums instead of needing all the datums to be
+successful, Pachyderm can mark some datums as *recovered* which means
+that they failed with a non-critical error, but the successful datums
+will be processed.
+
+To configure a condition under which you want your failed datums not
+to fail the whole job, you can add your custom error code in
+`err_cmd` and `err_stdin` fields in your pipeline specification.
+
+For example, your DAG consists of two pipelines:
+
+* The pipeline 1 cleans the data.
+* The pipeline 2 trains your model by using the data from the first pipeline.
+
+That means that the second pipeline takes the results of the first pipeline
+from its output repository and uses that data to train a model. In some cases,
+you might not need all the datums in the first pipeline to be successful
+to run the second pipeline.
+
+The following diagram describes how Pachyderm transformation and error
+code work:
+
+![err_cmd logic](../assets/images/err_cmd_workflow.svg)
+
+Here is what is happening in the diagram above:
+
+1. Pachyderm executes the transformation code that you defined in
+the `cmd` field against your datums.
+1. If a datum is processed without errors, Pachyderm marks it as
+`processed`.
+1. If a datum fails, Pachyderm marks executes your
+error code (`err_cmd`) on that datum.
+1. If the code in `err_cmd` successfully runs on the *skipped* datum,
+Pachyderm marks the skipped datum as `recovered` and marks the job as
```

Comment:
```suggestion
Pachyderm marks the skipped datum as `recovered`. That datum is still a failure and will not show up in the output, but it will not prevent successful datums from continuing onto the next step.
```